content/docs/setup/kubernetes/install/platform/ibm/index.md

```diff
@@ -52,18 +53,20 @@ This guide installs the current release version of Istio.
 {{< image link="./istio-catalog-2.png" caption="IBM Cloud Private - Istio Catalog" >}}
 
 - (Optional) Change the Istio version using the `CHART VERSION` drop-down.
-- Click the `Configure` button.
+- Click the `Configure` button at the bottom right corner.
 
 {{< image link="./istio-installation-1.png" caption="IBM Cloud Private - Istio Installation" >}}
 
-- Input the Helm release name (e.g. `istio-1.0.3`) and select `istio-system` as the target namespace.
-- Agree to the license terms.
+- Input the Helm release name (e.g. `istio`).
+- Select `istio-system` as the target namespace.
+- Select `local-cluster` as target cluster.
+- Check the agree checkbox after reading the license terms.
```

Comment:
`If you agree with the license terms, check the agree to license terms box`.  Otherwise, you are not providing a choice...

Reply:
Done